runtime/j9vm/j9vmnatives.xml

```diff
@@ -334,6 +334,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JVM_AreNestMates"/>
 		<export name="JVM_InitClassName"/>
 		<export name="JVM_InitializeFromArchive"/>
+
+		<!-- Additions for Java 14 (General) -->
+		<export name="JVM_GetExtendedNPEMessage"/>
```

Comment:
This needs to be in a new `<exports group="jdk14">` which can be conditionally referenced in the main and redirector modules.